packages/@aws-cdk/aws-pipes-targets-alpha/README.md

```diff
@@ -171,3 +172,36 @@ const pipe = new pipes.Pipe(this, 'Pipe', {
     target: pipeTarget
 });
 ```
+
+### Amazon EventBridge API Destination
+
+An EventBridge API destination can be used as a target for a pipe. 
+The API destination will receive the (enriched/filtered) source payload.
+
+```ts
+declare const sourceQueue: sqs.Queue;
+declare const dest: events.ApiDestination;
+
+const apiTarget = new targets.ApiDestinationTarget(dest);
+
+const pipe = new pipes.Pipe(this, 'Pipe', {
+    source: new SomeSource(sourceQueue),
```

Comment:
could you please use the correct Class name, this should be `SqsSource` instead of `SomeSource`

Reply:
The rest of the README has `SomeSource` but happy to change it.

Reply:
Fixed in https://github.com/aws/aws-cdk/pull/30756/commits/a4f52b2454ae2be4666288b20d4b53ced0bedfe3.